.gitattributes

```diff
@@ -0,0 +1,11 @@
+.appveyor.yml export-ignore
```

Comment:
What do you think about

- adding trailing slashes to distinguish between files and directories
- keeping entries also sorted by kind
- aligning `export-ignore`

?

```diff
diff --git a/.gitattributes b/.gitattributes
index 0abf5af7b4..fa872b5135 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,11 +1,11 @@
-.appveyor.yml export-ignore
-.editorconfig export-ignore
-.composer export-ignore
-.gitattributes export-ignore
-.github export-ignore
-.gitignore export-ignore
-.php_cs.dist export-ignore
-.travis.yml export-ignore
-build export-ignore
-phpunit export-ignore
-phpunit.xml.dist export-ignore
+.composer/          export-ignore
+.github/            export-ignore
+build/              export-ignore
+.appveyor.yml       export-ignore
+.editorconfig       export-ignore
+.gitattributes      export-ignore
+.gitignore          export-ignore
+.php_cs.dist        export-ignore
+.travis.yml         export-ignore
+phpunit             export-ignore
+phpunit.xml.dist    export-ignore
```

Reply:
agree with sorting, but not with aligning, for the same reason we don't align assigments: adding a new longer line would make us edit all lines.